dbflute-runtime/src/main/java/org/dbflute/cbean/coption/LikeSearchOption.java

```diff
@@ -55,6 +56,7 @@
     protected List<String> _originalWildCardList;
     protected List<SpecifiedColumn> _compoundColumnList;
     protected List<Integer> _compoundColumnSizeList;
+    protected boolean _isNullcompoundedAsEmpty;
```

Comment:
Frequently an instance variable name does not have prefix 'is'.
For example, _asOrSplit in LikeSearchOption.
An accessor method has it instead like isAsOrSprlit().
So rename to _nullCompoundedAsEmpty.

Reply:
> Frequently an instance variable name does not have prefix 'is'.
> For example, _asOrSplit in LikeSearchOption.
> An accessor method has it instead like isAsOrSprlit().
> So rename to _nullCompoundedAsEmpty.

It make sense. I understand very well.
https://github.com/dbflute/dbflute-core/pull/54/commits/af6b6ecf2249a76b9e15475f082f5f691c8024e0